modules/identity-shiro/build.gradle

```diff
@@ -0,0 +1,66 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+apply plugin: 'opensearch.internal-cluster-test'
+
+opensearchplugin {
```

Comment:
I think this module will be an ideal candidate for `sandbox` ?

Reply:
I believe there are 2 parts to it:
 - the core support (new plugin interfaces), this could be good candidate for future flags
 - the new experimental plugin (like this one), this could go to sandbox

Reply:
👍 I'll move the module into sandbox

Reply:
> sandbox changes the build output which is easy to test locally, but difficult from the distribution channels

@peternied Feel free to weigh in on #3677